prow/Makefile

```diff
@@ -1,16 +1,16 @@
 .phony: update-config update-plugins validate
 
 update-config:
-	kubectl create configmap config --from-file=config.yaml=config/config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
+	kubectl create configmap config --from-file=config.yaml=config/config.yaml --dry-run=client -o yaml | kubectl replace configmap config -f -
```

Comment:
curious to know why dry-run?

Reply:
dry run it to generate the configmap but it doesn't apply/replace it. We apply it afterwards with `| kubectl replace configmap config -f -`